src/index.ts

```
@@ -178,6 +194,27 @@ async function main() {
         return;
       }
 
+      const config = configManager.getConfig(guild.id);
+
+      // Conditionally register quest commands based on config
+      const questsEnabled = config.quests?.enabled !== false;
+      const questCommands = (client as any).questCommands || [];
+      
+      if (questsEnabled && questCommands.length > 0) {
```

Biome (2.3.14)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)